src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -72,7 +72,7 @@ export class GraphStyles {
 
             // use the workload name unless app label was defined
             let content = workload;
-            if (getCyGlobalData(ele).graphType !== GraphType.WORKLOAD && (app || app !== 'unknown')) {
+            if (getCyGlobalData(ele).graphType !== GraphType.WORKLOAD && app && app !== 'unknown') {
```

Comment:
This is better but I'm not sure it's correct.  I'll still merge.  But if graphType is APP then I think you should use 'app' in all cases.